test/e2e/specs/editor/various/copy-cut-paste.spec.js

```diff
@@ -565,6 +565,30 @@ test.describe( 'Copy/cut/paste', () => {
 		] );
 	} );
 
+	test( 'should paste link to formatted text', async ( {
+		page,
+		pageUtils,
+		editor,
+	} ) => {
+		await editor.insertBlock( {
+			name: 'core/paragraph',
+			attributes: { content: 'test' },
```

Comment:
Perhaps it could start with bolded text to reduce the complexity of the test?

Reply:
Indeed, I simplified it with 9a56955